src/sections/Home/Banner/index.js

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect, useState } from "react";
+import BannerDefault from "../Banner-default";
+// import BannerAlt from "../Banner-alt"; 
+
+const Banners = [
+  <BannerDefault key={0}/>,
+  // <BannerAlt key={1}/> 
+];
+
+const Banner = () => {
+  const [Banner, setBanner] = useState(sessionStorage.getItem("banner") || 0);
```

Comment:
shouldn't it be `window.sessionStorage`?

Reply:
sessionStorage refers to window.sessionStorage. We can simply use sessionStorage if no other variable of the same name exists.

Reply:
Yea, you're right. Thanks! :smile: